### PR TITLE
Implement officer legacyPicture workaround

### PR DESCRIPTION
**Pull request by diamondburned**
This commit implements a workaround to fit board members with legacy
pictures onto the new <BoardPicture> component. It does this with
hard-coded translations to overlap the new frame onto the old pictures.

This hard-coding is written with the expectation that no legacy pictures
will be added in the future. It's not guaranteed to work with new legacy
pictures.

### CODE REVIEW

File: src/lib/components/board-member/board-picture.svelte [hunk 1]
```diff
@@ -4,6 +4,8 @@
    */
   export let src: string;
   export let color: string;
+  export let legacy = false;
+  console.log(src, legacy);
```
Comment (EthanThatOneKid) [inline src/lib/components/board-member/board-picture.svelte]: Please remove unintended console logs before merging.

File: src/lib/components/board-member/board-picture.svelte [hunk 2]
```diff
@@ -30,8 +36,16 @@
...
+  img {
     width: 100%;
     height: 100%;
+
+    &.legacy {
+      width: calc(155%);
+      height: calc(155%);
+      transform: translate(-17.75%, -14.5%);
+      /* Uncomment for debugging: */
+      /* background-color: red; */
```
Comment (EthanThatOneKid) [inline src/lib/components/board-member/board-picture.svelte]: Remove debugging code.

**Review by EthanThatOneKid — APPROVED** [review]
Most importantly, the resulting `src/routes/(site)/about/officer-profile.svelte` file looks good to me. I left a few comments to resolve before merging.

File: src/lib/components/board-member/board-picture.svelte (hunk 1 above)
Comment (diamondburned) [reply src/lib/components/board-member/board-picture.svelte]: oops

File: src/lib/components/board-member/board-picture.svelte [hunk 3]
```diff
@@ -15,7 +16,11 @@
       />
     </clipPath>
   </defs>
-  <image clip-path="url(#shape)" xlink:href={src} />
+  <foreignObject x="0" y="0" width="100%" height="100%" clip-path="url(#shape)">
+    <div xmlns="http://www.w3.org/1999/xhtml">
```
Comment (EthanThatOneKid) [inline src/lib/components/board-member/board-picture.svelte]: I'd recommend resolving the lint errors. [Here is one](https://github.com/EthanThatOneKid/acmcsuf.com/actions/runs/6089638664/job/16522854449#step:7:55):

```
/home/runner/work/acmcsuf.com/acmcsuf.com/src/lib/components/board-member/board-picture.svelte:20:10
Error: Argument of type '{ xmlns: string; }' is not assignable to parameter of type 'HTMLProps<"div", HTMLAttributes<any>>'.
  Object literal may only specify known properties, and '"xmlns"' does not exist in type 'HTMLProps<"div", HTMLAttributes<any>>'. (ts)
  <foreignObject x="0" y="0" width="100%" height="100%" clip-path="url(#shape)">
    <div xmlns="http://www.w3.org/1999/xhtml">
      <img {src} alt="Board member" class:legacy />


====================================
svelte-check found 1 error and 0 warnings in 1 file

Error: Process completed with exit code 1.
```

Comment (EthanThatOneKid) [reply src/lib/components/board-member/board-picture.svelte]: Resolved in <https://github.com/EthanThatOneKid/acmcsuf.com/pull/899/commits/5a686d7522a4853baf2b49a001877295ff3bc12e>.

**Pull request merged**